ansible/roles/routemgmt/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+apigw_auth_user: ""
+apigw_auth_pwd: ""
+apigw_host: "http://{{ groups['edge']|first }}:3100/v1"
```

Comment:
This defaults file can now be deleted.  Based on http://docs.ansible.com/ansible/playbooks_variables.html#id31 it seemed to provide default values that could be overridden by other playbook variable settings.  But as builds progressed, these default values were not as pervasive as I had assumed.